### ISSUE TITLE
Volume Boost should have a separate control

### ISSUE THREAD

**Issue by Khuerustom** [bitgapp/eqMac#358]
While increasing the volume with keyboard, the volume passes 100% and boost increases afterwards. There should be a control switch so that the user can choose if they want boost after 100% volume. Another thing that could be done is boost should have a separate slider independent from volume control.

**Comment by nodeful** [comment]
No it shouldn't. Thanks for you opinion :D

**Comment by andresilva-cc** [comment]
> No it shouldn't. Thanks for you opinion :D

Well, that's very rude of you. Of course, the software is yours and you do whatever you want, but what @Khuerustom suggested is a really nice touch on user experience as not everybody expects the volume to work that way, myself included.

For example, I just discovered after some days using eqMac that the boost, by default, was a little higher, and some songs with a lot of bass were distorting, so I set the volume to 100% (without boost) and it solved the issue. But now whenever I press volume up to see if it's already on max, it activates boost.

There's another [issue](https://github.com/bitgapp/eqMac/issues/427) about this too, I left a thumbs up there.
